app/api/domains/cho.py

```diff
@@ -759,12 +759,21 @@ async def handle_osu_login_request(
 
     # TODO: store adapters individually
 
+    # some disk manufacturers have "0" for their disk serial, resulting in clashing md5 hashes of the string "0".
+    # these need to be ignored since otherwise people will be prevented from verifying a lot.
+    INACTIONABLE_DISK_SIGNATURE_MD5S = ["dcfcd07e645d245babe887e5e2daa016"]
```

Comment:
This is fine for now imo -- comments regarding having a db table or config file are indeed more flexible, so I'd be open to a follow-up PR